src/tools/tidy/src/ui_tests.rs

```diff
@@ -7,7 +7,7 @@ use std::path::Path;
 
 const ENTRY_LIMIT: usize = 1000;
 // FIXME: The following limits should be reduced eventually.
-const ROOT_ENTRY_LIMIT: usize = 982;
+const ROOT_ENTRY_LIMIT: usize = 983;
```

Comment:
Because a new folder was added to accommodate the tests for this feature and `src/test/ui` currently can not exceed more than 982 entries.

This change is funny because I was trimming down the whole test suite -> https://github.com/rust-lang/rust/issues/73494

@petrochenkov Can we please remove `ROOT_ENTRY_LIMIT` to keep it in agreement with the remaining folders (1000 entries max)? Full separation of standalone unit test files into appropriated directories can be performed afterwards and does not depend on `ROOT_ENTRY_LIMIT`

Reply:
I would prefer to move the whole `rfc-3086-metavar-expr` subdirectory to `ui/macros`.

Reply:
Moved to `ui/macros` but I still insist on removing `ROOT_ENTRY_LIMIT` in another PR because people will simply keep increasing the value and it probably won't take long to exceed 1000 entries (again). Besides, its existence was due to its large amount of files which is not the case anymore.

`ui/parser` is also nearing 1000 entries so IMO, all directories should be constantly monitored and fragmented over time without any special handling.